faq/new-object-storage-classes.mdx

```diff
@@ -10,10 +10,11 @@ hero: assets/objectstorage.webp
 
 <Concept>
 
-## What is the difference between the Standard class in PAR and the Standard class in AMS and WAW?
+## What is the difference between the Standard class in PAR and AMS, and the Standard class in WAW?
 Scaleway Object Storage uses erasure code (6+3) to secure your data, today the data is distributed across three different racks inside the same DC. In case one rack becomes unavailable, we are still able to rebuild your data. 
```

Comment:
```suggestion
Scaleway Object Storage uses erasure code (6+3) to secure your data, which is currently distributed across three different racks inside the same DC. In case one rack becomes unavailable, we are still able to rebuild your data. 
```